packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -137,4 +151,5 @@ export interface Lb3AppBooterOptions {
 
 interface Lb3Application extends ExpressApplication {
   handler(name: 'rest'): ExpressRequestHandler;
+  dataSources?: AnyObject;
```

Comment:
Can we use a stricter type definition please?

For example:

```suggestion
  dataSources?: {[name: string}: object};
```

Or perhaps:

```suggestion
  dataSources?: {[name: string}: {name: string}};
```

Reply:
Going with `dataSources?: {[name: string]: DataSource};`.